core/test/unit/services/url/utils_spec.js

```diff
@@ -438,14 +438,16 @@ describe('Url', function () {
             urlService.utils.urlFor('admin', true).should.equal('http://something.com/blog/ghost/');
         });
 
-        ['deprecated', 'active'].forEach((apiVersion) => {
+        ['deprecated', 'active', 'v0.1', 'v2'].forEach((apiVersion) => {
```

Comment:
Hmmm adding v0.1 means, we call in the test

> .urlFor('api', {version: v0.1, versionType: 'content'}, true)

right?

Reply:
Yeah correct. Basically the change allows to use either one of the supported versions by `stable`/`active`/`deprecated` or directly by that particular version name.

Reply:
The reason i asked is, that `.urlFor('api', {version: v0.1})` is not tested, or?

Reply:
Oh yeah correct its not tested, all tests were explicit with version type so far. I'll add a separate test for this case 👍

Reply:
Cool. Feel free to self merge afterwards 👍